masterlist.yaml

```
@@ -13555,21 +13555,28 @@ plugins:
         <<: *dirtyPlugin
         itm: 5
         udr: 3
+
   - name: 'Blood&Mud.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/12016' ]
-    tag:
-      - Hair
-      - NpcFaces
     msg:
       - <<: *wildEdits
-        subs: [ 'Record names have been changed, you can use the xEdit script **Restore record names from master** to correct them.' ]
-        condition: 'checksum("Blood&Mud.esp", 23A62955)'
+        subs: [ 'Incorrectly renames various records, use the **Restore record names from master** TES4Edit script to fix them.' ]
+        condition: 'checksum("Blood&Mud.esp", 23A62955) or checksum("Blood&Mud.esp", E9B56954)'
```

@Infernio Don't forgot to change the post-cleaning checksum 😛 

```suggestion
        condition: 'checksum("Blood&Mud.esp", 23A62955) or checksum("Blood&Mud.esp", FAC5765D)'
```

---

🤦‍♀️